src/Event/Events/EventFromUDB2.php

```diff
@@ -151,8 +152,8 @@ private function getCalendar(array $calendarAsArray): Calendar
                     $startTime;
             $subEvents[] = new SubEvent(
                 new DateRange(
-                    \DateTimeImmutable::createFromFormat('Y-m-d\TH:i:s', $startTime, new DateTimeZone('Europe/Brussels')),
-                    \DateTimeImmutable::createFromFormat('Y-m-d\TH:i:s', $endTime, new DateTimeZone('Europe/Brussels'))
+                    DateTimeFactory::fromFormat('Y-m-d\TH:i:s', $startTime, new DateTimeZone('Europe/Brussels')),
```

Comment:
Denk dat je hiervoor de core constant DATE_ISO8601 kan gebruiken. (https://www.php.net/manual/en/datetime.constants.php#constant.date-iso8601)

Reply:
As discussed best to fix in dedicated ticket